src/cloudai/systems/slurm/slurm_command_gen_strategy.py

```
@@ -105,6 +106,147 @@ def container_mounts(self) -> list[str]:
+                "set -euo pipefail",
+                f"source_hub={source_hub}",
+                f"target_hf_home={target_hf_home}",
+                'target_hub="$target_hf_home/hub"',
+                'staging_state="$target_hf_home/.cloudai-staging"',
+                'command -v flock >/dev/null 2>&1 || { echo "CloudAI: local HF staging requires flock" >&2; exit 1; }',
+                "cache_manifest() {",
+                "  {",
+                "    find \"$1\" \\( -type f -printf 'f %P %s\\n' -o -type l -printf 'l %P %l\\n' \\)",
+                '    if [ -d "$1/refs" ]; then',
+                "      find \"$1/refs\" -type f -printf '%P\\0' | LC_ALL=C sort -z | "
+                "while IFS= read -r -d '' ref_path; do",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)